### DISCUSSION TITLE
Everyone should be able to add labels (idea 💡)

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#14624] [Projects and Issues]
Selam. (Hi.)

Herkes etiket eklemeli diye düşünüyorum. (I think everyone should add tags.) 
Peki her etiketi ekleyebilir mi? Hayır! (So can add every tag? No!)
Öyleyse etiket ayarından izin verelim. (So let's let from the tag setting.) 👊

![resim](https://user-images.githubusercontent.com/25375593/162577585-c454b961-191e-4117-a544-4185c450b48b.png)

**Comment by queenofcorgis** [comment]
Hi there - where would you like to be able to add labels?

**Reply by volkantash** [reply]
👋
Ben en başta tartışmalardaki başlıklar ve sızılardaki başlıklar için etiket eklemek isterim. (First of all, I would like to add labels for the topics in the discussions and the topics in the issue.)

Bu etiketleri repo sahipleri yaratmalı. Ve etiketleri başlıklara ekleme izinlerini ayarlamalıdır. Kullanıcı, takım, herkes. (These tags should be created by the repo owners. And it should set permissions to add tags to headers. The user, the team, everyone.)

**Ek fikir: (Additional idea:)**
Hiç olmayan bir etiketi herhangi bir kullanıcı oluşturmak isteyebilir. Bence gereksiz. Ama gerekli bir yanı var ise insanları çıldırtmayacak bir etiket yaratıp ekleme özelliği sunabilirsiniz. (Any user may want to create a non-existent tag. I think it's unnecessary. But if there is a necessary aspect, you can create and add a tag that will not drive people crazy.)
